internal/functions/deploy/deploy_test.go

```diff
@@ -0,0 +1,87 @@
+package deploy
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/supabase/cli/internal/testing/apitest"
+	"github.com/supabase/cli/internal/utils"
+	"github.com/supabase/cli/pkg/api"
+	"gopkg.in/h2non/gock.v1"
+)
+
+func init() {
+	// Setup fake deno binary
+	if denoPath, err := os.Executable(); err == nil {
+		utils.DenoPathOverride = denoPath
+	}
+}
+
+func TestMain(m *testing.M) {
+	if len(os.Args) > 1 && (os.Args[1] == "bundle" || os.Args[1] == "upgrade") {
+		fmt.Println("ea09bff03b3894b79e335f77a0f71d75adf263e9")
```

Comment:
What's the magic number?

Reply:
Can be deleted. Was using it to check stdout.